### ISSUE TITLE
Create JUG index page

### ISSUE THREAD

**Issue by cesarhernandezgt** [wwjugs/Global#5]
Add a JUG index page order alphabetically by Country.
For example, the `jugIndex` will contain a reference to  `git/jugs/Global/JUGS/M/`

Taking into consideration that one Country can have 1-*(to many) JUGs, the proposal for the tree structure is:

```
.
└── Global
    ├── JUGS
    │   ├── E
    │   │   └── Espain
    │   ├── G
    │   │   └── Guatemala
    │   ├── M
    │   │   └── Mexico
    │   │       ├── JUG-1
    │   │       └── JUG-2
    │   └── aTemplate.md
    ├── LICENSE
    ├── README.md
    └── jugIndex.md
```

**Comment by cesarhernandezgt** [comment]
As for March 1st 2021, the homepage has the entire list of JUGS, I like the idea of having a collapsable rendering in alphabetical order. I'll keep this issue open in case this idea makes sense in the future when the home page gets too long.

**Comment by FDelporte** [comment]
Or instead of collapsable, a new page with a full (searchable? orderable?) table with all the JUGs? Just an idea...
https://www.w3schools.com/howto/howto_js_filter_table.asp

**Comment by icougil** [comment]
I agree with @FDelporte probably a lookup table will be much easier, and useful I guess, in the end, everyone who comes to the page,... what they will probably do is to search for a particular JUG, right? 🤔
